core/templates/dev/head/base.html

```diff
@@ -56,7 +56,7 @@
 
     <link rel="apple-touch-icon" href="/images/logo/favicon.png">
     <title itemprop="name">{% block subtitle %}{% endblock %} - {% block maintitle %}Oppia{% endblock %}</title>
-
+    <base href="/gallery">
```

Comment:
Hm, I'm not too worried (yet) about where it gets declared. I'm mostly worried because it doesn't seem to work properly.